### PR TITLE
[MM-55110] Replace usage of LocalizedIcon in 'custom_theme_chooser.tsx' with i/span tags

### PR DESCRIPTION
**Pull request by PulkitGarg-code**
#### Summary

In the PR, I have replaced the usage of LocallizedIcon tag to <i> tag and also replaced the usage of t() as its not with accordance with react-intl.

https://github.com/mattermost/mattermost/issues/25100

#### Ticket Link

https://github.com/mattermost/mattermost/issues/25100

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/user_settings/display/user_settings_theme/custom_theme_chooser/custom_theme_chooser.tsx [hunk 1]
```diff
@@ -289,8 +287,9 @@ export default class CustomThemeChooser extends React.PureComponent<Props, State
     };
 
     render() {
-        const theme = this.props.theme;
+        const {formatMessage} = useIntl();
```
[the remark this answers is not in the document]
Comment (PulkitGarg-code) [reply webapp/channels/src/components/user_settings/display/user_settings_theme/custom_theme_chooser/custom_theme_chooser.tsx]: i have resolved this, once check

**Comment by M-ZubairAhmed** [comment]
@PulkitGarg-code it looks like you haven't sign the CLA
please follow this https://github.com/mattermost/mattermost/pull/25537#issuecomment-1826234729

File: webapp/channels/src/components/user_settings/display/user_settings_theme/custom_theme_chooser/custom_theme_chooser.tsx [hunk 2]
```diff
@@ -289,8 +287,9 @@ export default class CustomThemeChooser extends React.PureComponent<Props, State
     };
 
     render() {
-        const theme = this.props.theme;
+        const {formatMessage} = this.props;
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/user_settings/display/user_settings_theme/custom_theme_chooser/custom_theme_chooser.tsx:290]: Oops we deleted the `theme` variable here
```
const {formatMessage, theme} = this.props
```

File: webapp/channels/src/components/user_settings/display/user_settings_theme/custom_theme_chooser/custom_theme_chooser.tsx [hunk 3]
```diff
@@ -552,3 +551,5 @@ export default class CustomThemeChooser extends React.PureComponent<Props, State
         );
     }
 }
+
+export default injectIntl(CustomThemeChooser);
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/user_settings/display/user_settings_theme/custom_theme_chooser/custom_theme_chooser.tsx:555]: empty line after this

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Please check the failing CI and rerun the test to see why they are failing, you might have to run update snapshots on these

**Comment by M-ZubairAhmed** [comment]
@PulkitGarg-code the issue https://github.com/mattermost/mattermost/issues/25093 was already done and merged, so we can focus on https://github.com/mattermost/mattermost/issues/25100 in this PR

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
The code looks really good. Can you please try to resolve the errors in the CI failures?

**Comment by PulkitGarg-code** [comment]
Hi @M-ZubairAhmed, almost all the checks have passed. please review and let me know if anything is required from my side to merge this PR.

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**